lib/game.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require_relative './Board'
+require_relative './UI'
+require_relative './game_logic'
+
+# this class will wrap the board, UI together
+class Game
+  attr_reader :board, :ui, :game_logic
+  
+  def initialize
+    @board = create_new_board
+    @ui = create_new_ui
+    @game_logic = create_game_logic
+  end
```

Comment:
Maybe you can clean this up some by using dependency injection... 
```suggestion
  def initialize(board = Board.new, ui = Ui.new, game_logic = GameLogic.new)
    @board = board
    @ui = ui
    @game_logic = game_logic
  end
```